tests/wamr-test-suites/test_wamr.sh

```diff
@@ -58,7 +59,7 @@ QEMU_FIRMWARE=""
 # prod/testsuite-all branch
 WASI_TESTSUITE_COMMIT="cf64229727f71043d5849e73934e249e12cb9e06"
 
-while getopts ":s:cabgvt:m:MCpSXxwPGQF:" opt
+while getopts ":s:cabgvt:m:MCpSXxwPGQFj:" opt
```

Comment:
Should not change "F:" to "F", ':' means there is an extra argument for the the option, e.g. "-F <QEMU_FIRMWARE>" and "-m \<TARGET\>". So here it should be "F:j".

How about using "-j <PLATFORM>" to specify a platform so we could support more platforms in the future? In which platform is set in L55 "PLATFORM=$(uname -s | tr A-Z a-z)" by default, and we can use "./test_wamr.sh -j cosmopolitan" to set it to cosmopolitan.
Here change it to "F:j:", and change below to be like:
```shell
        j)
        echo "test platform " ${OPTARG}
        PLATFORM=${OPTARG}
        ;;
```

Reply:
Good catch and suggestion. Fixed`-F` and adapted `-j` to specify platform in a3449ac .

Reply:
Adjusted spacing to match in 27023ee